src/cascadia/TerminalSettingsEditor/Appearances.xaml

```diff
@@ -215,12 +205,12 @@
                                     SettingOverrideSource="{x:Bind Appearance.BackgroundImagePathOverrideSource, Mode=OneWay}">
                 <StackPanel Orientation="Vertical">
                     <StackPanel Orientation="Horizontal">
-                        <TextBox IsEnabled="{x:Bind Appearance.BackgroundImagePath, Mode=OneWay, Converter={StaticResource StringIsNotDesktopConverter}}"
+                        <TextBox IsEnabled="{x:Bind local:Converters.StringsAreNotEqual('desktopWallpaper', Appearance.BackgroundImagePath), Mode=OneWay}"
                                  Style="{StaticResource TextBoxSettingStyle}"
-                                 Text="{x:Bind Appearance.BackgroundImagePath, Mode=TwoWay, Converter={StaticResource DesktopWallpaperToEmptyStringConverter}}" />
+                                 Text="{x:Bind local:Converters.StringFallBackToEmptyString('desktopWallpaper', Appearance.BackgroundImagePath), Mode=TwoWay, BindBack=Appearance.SetBackgroundImagePath}" />

```

Comment:
okay so this TextBox is enabled when the `BackgroundImagePath != 'desktopWallpaper'`. When the `BackgroundImagePath=='desktopWallpaper'`, `StringFallBackToEmptyString` will convert that to `desktopWallpaper`. When the string is anything else, it'll bind the text as the empty string? Is this right? This might need some comments adjacent to it to help explain what's going on here (though admittedly I didn't leave any behind when [I wrote this the first time](https://github.com/microsoft/terminal/pull/8778/files#diff-fa9becdd808a689c0f1029f17d57635ea935099faee1eb4f3832c6c30acbe576) 😋)

Reply:
I think right now we're actually filling the box with the empty string, but still enabling the rest of background image settings. This PR updates it to actually fill the box with `desktopWallpaper`, which is a little weird, but probably okay? Everything else still seems to work fine.